src/app/datagrid/utils/pokemon-comparator.ts

```diff
@@ -3,11 +3,12 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Comparator} from "../../../clarity-angular/datagrid";
+
+import { Comparator } from "../../../clarity-angular/data/datagrid/interfaces/comparator";
```

Comment:
I don't remember these imports for the demos being so complex, but either way we should just import from `"../../../clarity-angular"`. The `index.ts` file exports all these.

It's not a blocker, this PR can go through as is, but you made the imports even worse than before, which is why I suddenly noticed it.